doc/sphinx-guides/source/admin/integrations.rst

```diff
@@ -77,6 +85,41 @@ SampleDB is a web-based electronic lab notebook (ELN) with a focus on flexible m
 
 For instructions on using the Dataverse export, you can visit https://scientific-it-systems.iffgit.fz-juelich.de/SampleDB/administrator_guide/dataverse_export.html
 
+RedCap
+++++++
+
+RedCap is a web-based application to capture data for clinical research and create databases and projects.
+
+The :ref:`integrations-dashboard` enables a pull of data from RedCap to a dataset in Dataverse.
+
+GitLab
+++++++
+
+GitLab is an open source Git repository and platform that provides free open and private repositories, issue-following capabilities, and wikis for collaborative software development.
+
+The :ref:`integrations-dashboard` enables a pull of data from GitLab to a dataset in Dataverse.
+
+iRODS
++++++
+
+An open source, metadata driven data management system that is accessible through a host of different clients.
+
+The :ref:`integrations-dashboard` enables a pull of data from iRODs to a dataset in Dataverse.
```

Comment:
"iRODs" -> "iRODS"

Reply:
@trel thanks! Fixed in 304bca6a6d4c2b215e348ee79066dda1bd7cfd9d.